src/Chapter1.hs

```diff
@@ -652,8 +667,11 @@ Implement a function that returns the first digit of a given number.
 You need to use recursion in this task. Feel free to return to it later, if you
 aren't ready for this boss yet!
 -}
-
-firstDigit n = error "firstDigit: Not implemented!"
+firstDigit :: Int -> Int
+firstDigit n = if divN == 0 then mod absN 10 else firstDigit(divN)
```

Comment:
Great solution!

If `divN == 0` that exactly means that the number is < 10, so you don't need to do `mod` on that, as it shouldn't change the number 🙂